src/Compilers/Core/Portable/Compilation/NullableContextOptions.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#nullable enable
+using System;
+
+namespace Microsoft.CodeAnalysis
+{
+    /// <summary>
+    /// Represents the default state of nullable analysis in this compilation.
+    /// </summary>
+    [Flags]
+    public enum NullableContextOptions
+    {
+        /// <summary>
+        /// The nullable analysis feature is disabled.
+        /// </summary>
+        Disable = 0b00,
+
+        /// <summary>
+        /// Nullable warnings are enabled and will be reported by default.
+        /// </summary>
+        Warnings = 0b01,
+
+        /// <summary>
+        /// Nullable annotations are enabled and will be shown when APIs
+        /// defined in this project are used in other contexts.
+        /// </summary>
+        Annotations = 0b10,
+
+        /// <summary>
+        /// The nullable analysis feature is fully enabled.
+        /// </summary>
+        Enable = Warnings | Annotations,
```

Comment:
"All"?  And "None" for 'Disable'?

Reply:
These are names that will be used in source and on the command line, so we want to be consistent.